Apps/DiceRoller/DiceRoller/ISDRProjectManager.h

```diff
@@ -32,6 +32,7 @@
 @property (weak, nonatomic) IBOutlet UIButton *closeBtn;
 - (IBAction)closeBtnOnClick:(id)sender;
 
+
 -(bool)projectHasValidFields;
```

Comment:
Two spaces between `- (IBAction)closeBtnOnClick:(id)sender;` and ` -(bool)projectHasValidFields;` right now - let's make it one.